src/main/java/reposense/util/FileUtil.java

```diff
@@ -168,6 +169,18 @@ public static void createDirectory(Path dest) throws IOException {
         Files.createDirectories(dest);
     }
 
+    /**
+     * Returns true if {@code path} is a valid path.
+     */
+    public static boolean isValidPath(String path) {
```

Comment:
There is a more exhaustive `isValidPath` done by one of your seniors in the past.
Please check if its still relevant https://github.com/reposense/RepoSense/commit/3157e1b204ebbd1f2e9693d1d8af24423bcd6624

Technically if this new implementation can prevent that issue, then we should remove all the deprecated codes.

Reply:
I believe the one done up in this commit https://github.com/reposense/RepoSense/commit/3157e1b204ebbd1f2e9693d1d8af24423bcd6624 serves a different purpose than as compared to this implementation.

The implementation of `isValidPath` in this PR only deals with files in the repo that contain illegal characters (typically in Windows OS), ensuring that the file name is compatible with whatever the user's OS is, whereas the one in https://github.com/reposense/RepoSense/commit/3157e1b204ebbd1f2e9693d1d8af24423bcd6624 deals with globs.

Reply:
I see. Can you please rename that to isValidIgnoreGlob, with the `path` rename to `IgnoreGlob` as well, to avoid any possible conflict of apis/misleadings in future?

E.g. `private static boolean isValidIgnoreGlob(File repoRoot, String ignoreGlob) {`

Reply:
Ok, made the changes.